index.html

```diff
@@ -203,13 +212,15 @@ <h2 class="is-size-4" style="margin-bottom: 1rem;">Options</h2>
                         :existing-tags="[
                             { key: 'children-of-bodom', value: 'Children of Bodom' },
                             { key: 'epica', value: 'Epica' },
+                            { key: 'epica', value: 'Épica' },
```

Comment:
In real life "Epica" and "Épica" could be two different words with different meanings, right? So the keys should also be different so you would be able to select both words. Please fix your examples here or think of different words.

Reply:
Sure, I'll add just one little example of an accentuated word.